svAssistant.py

```diff
@@ -305,11 +271,12 @@ def set_type_seeds(season, date):
 
 def per_day_income(crop_name, date):
     """
-    Given the name of the crop and the current date, this will return the amount
-    of gold that the crop can produce if harvested until the end of its life
-    span.
+    Given the name of the crop and the current date, this will return the
+    amount of gold that the crop can produce if harvested until the end of its
+    life span.
     """
-    # If-block triggered if the crop is a multi-harvestable crop (e.g. strawberries)
+    # If-block triggered if the crop is a multi-harvestable crop
+    # (e.g. strawberries)
     if crop_name in multiharvest_crops:
```

Comment:
An entire function that's just a giant `if`/`else` with no shared logic between them is a pretty strong sign that these should be two separate functions that get called from the `if`/`else`.